### PR TITLE
Latex Fixes in crypto.py

### PR DESCRIPTION
**Pull request by sudoWin**
#### References to other Issues or PRs

Adds Latex to Function in crypto.py
https://docs.sympy.org/latest/modules/crypto.html#sympy.crypto.crypto.gm_private_key
#### Other comments

#18859 
#### Release Notes

NO ENTRY

### CODE REVIEW

**Comment by sylee957** [comment]
LaTeX should be denoted by `$$` instead of backticks from the recent docstring guide.

**Comment by sudoWin** [comment]
Changed to $ signs.

File: sympy/crypto/crypto.py [hunk 1]
```diff
@@ -2899,37 +2899,42 @@ def _random_coprime_stream(n, seed=None):
...
-        let a_k be a random square
+     if $b_k$ = 0:
+        let $a_k$ be a random square
         (quadratic residue) modulo p * q
-        such that jacobi_symbol(a, p * q) = 1
-     if b_k = 1:
-        let a_k be a random non-square
-        (non-quadratic residue) modulo p * q
-        such that jacobi_symbol(a, p * q) = 1
+        such that $jacobi \\_symbol(a, pq) = 1$
```
Comment (sylee957) [inline sympy/crypto/crypto.py]: It's better to have a spacing between `pq` even if it renders the same.

File: sympy/crypto/crypto.py [hunk 2]
```diff
@@ -2899,37 +2899,42 @@ def _random_coprime_stream(n, seed=None):
...
+    $\\cdot$ Given a message as an integer $i$, write i in its
+    bit representation $b_0$ , $\\dotsc$ , $b_n$ .
+
+    $\\cdot$ For each k,
 
-     if b_k = 0:
-        let a_k be a random square
+     if $b_k$ = 0:
+        let $a_k$ be a random square
         (quadratic residue) modulo p * q
```
Comment (sylee957) [inline sympy/crypto/crypto.py]: Why not change p * q to latex

File: sympy/crypto/crypto.py [hunk 3]
```diff
@@ -2899,37 +2899,42 @@ def _random_coprime_stream(n, seed=None):
...
     roughly as follows.
 
-    Pick two large primes p ands q. Call their product N.
-    Given a message as an integer i, write i in its
-    bit representation b_0,...,b_n. For each k,
+    $\\cdot$ Pick two large primes $p$ and $q$.
+
+    $\\cdot$ Call their product $N$.
+
+    $\\cdot$ Given a message as an integer $i$, write i in its
```
Comment (sylee957) [inline sympy/crypto/crypto.py]: Why not change the second i to LaTeX form?

File: sympy/crypto/crypto.py [hunk 4]
```diff
@@ -2899,37 +2899,42 @@ def _random_coprime_stream(n, seed=None):
...
+    bit representation $b_0$ , $\\dotsc$ , $b_n$ .
+
+    $\\cdot$ For each k,
```
Comment (sylee957) [inline sympy/crypto/crypto.py]: Why not change k to LaTeX form?

**Comment by sudoWin** [comment]
Fixed the above accidental leftovers

**Comment by smichr** [comment]
I added a couple of other small changes.

**Comment by sudoWin** [comment]
Thanks @smichr  
Do you know why wasn't flake8 showing anything for that unused keyword?

File: sympy/crypto/crypto.py [hunk 5]
```diff
@@ -2899,37 +2900,42 @@ def _random_coprime_stream(n, seed=None):
...
-    b_k can be recovered by checking whether or not
-    a_k is a residue. And from the b_k's, the message
+    $b_k$ can be recovered by checking whether or not
+    $a_k$ is a residue. And from the $b_k$ 's, the message
     can be reconstructed.
 
-    The idea is that, while jacobi_symbol(a, p * q)
-    can be easily computed (and when it is equal to -1 will
-    tell you that a is not a square mod p * q), quadratic
+    The idea is that, while $jacobi \\_ symbol(a, pq)$
```
Comment (smichr) [inline sympy/crypto/crypto.py]:
```suggestion
    The idea is that, while $jacobi \\_ symbol(a, p q)$
```

File: sympy/crypto/crypto.py [hunk 6]
```diff
@@ -2899,37 +2900,42 @@ def _random_coprime_stream(n, seed=None):
...
+    can be easily computed (and when it is equal to $-1$ will
+    tell you that $a$ is not a square mod $pq$), quadratic
```
Comment (smichr) [inline sympy/crypto/crypto.py]:
```suggestion
    tell you that $a$ is not a square mod $p q$), quadratic
```

File: sympy/crypto/crypto.py [hunk 7]
```diff
@@ -2899,37 +2900,42 @@ def _random_coprime_stream(n, seed=None):
...
     residuosity modulo a composite number is hard to compute
     without knowing its factorization.
 
-    Moreover, approximately half the numbers coprime to p * q have
-    jacobi_symbol equal to 1. And among those, approximately half
+    Moreover, approximately half the numbers coprime to $pq$ have
```
Comment (smichr) [inline sympy/crypto/crypto.py]:
```suggestion
    Moreover, approximately half the numbers coprime to $p q$ have
```

**Pull request merged**